Update Dockerfile

velvet-lynx opened the pull request:
Trying to solve last failing build by updating the react-scripts version

velvet-lynx committed:
Update Dockerfile

Trying to solve last failing build by updating the react-scripts version

sissou96 commented:
Checked on StackOverflow for a any known bug. It seems like a way to fix the issue is to add:
```
RUN npm config set unsafe-perm true
```
Will try on next commit

velvet-lynx committed:
Update Dockerfile

Another attempt to fix the build error issue on Travis CI.
Reverting back to previous react-scripts version and setting npm's unsage-perm to true.

velvet-lynx committed:
Typo on react-scripts version
